apps/examples/wc-vue3/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "wc-vue3",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "run-p type-check build-only",
```

Comment:
I was going to ask for this to be renamed so that it doesn't get built as part of the main monorepo build, but that doesn't seem to be happening for some reason (good news).
Do we know why this is? I thought it was to do with `"private": true` but if I remove that then it still doesn't get built... 🤔